readthedocs/rtd_tests/tests/test_core_tags.py

```diff
@@ -14,9 +14,13 @@
 class CoreTagsTests(TestCase):
     fixtures = ['eric', 'test_data']
 
+    @pytest.fixture(autouse=True)
```

Comment:
I dont think fixture works in unittest style tests. Can you remove the fixture?

Reply:
Yes It Doesn't Work on the unittest Style Tests but to remove the pytest warnings we have to use the request fixture to access `config.option.url_scheme ` as The `pytest.config` object is deprecated. Is there any better way to get the same result?